plugins/net_plugin/net_plugin.cpp

```diff
@@ -3928,15 +3936,18 @@ namespace eosio {
    }
 
    void net_plugin_impl::bcast_vote_message( uint32_t exclude_peer, const chain::vote_message_ptr& msg ) {
-      buffer_factory buff_factory;
-      auto send_buffer = buff_factory.get_send_buffer( *msg );
```

Comment:
Move the serialization of vote_message off of main thread and move it down below into the net thread pool.